tests/Builder/Type/CombinedFieldQueryTest.php

```diff
@@ -21,9 +24,31 @@ public function testEmptyFieldQueries(): void
     public function testFieldQueries(): void
     {
         $fieldQueries = new CombinedFieldQuery([
-            $this->createMock(CombinedFieldQuery::class),
+            new EqOperator(1),
             ['$gt' => 1],
-            new CombinedFieldQuery([]),
+            (object) ['$lt' => 1],
+            new class implements Serializable {
+                public function bsonSerialize(): array
+                {
+                    return ['$gte' => 1];
+                }
+            },
+        ]);
+
+        $this->assertCount(4, $fieldQueries->fieldQueries);
```

Comment:
I suppose asserting the output would have no effect as the constructor doesn't change the values in any way -- it just validates. Correct?

Reply:
Exact.